sklearn/inspection/tests/test_plot_partial_dependence.py

```diff
@@ -85,13 +85,29 @@ def test_plot_partial_dependence(grid_resolution, pyplot, clf_boston, boston):
     assert ax.get_ylabel() == boston.feature_names[1]
 
 
-def test_plot_partial_dependence_str_features(pyplot, clf_boston, boston):
+@pytest.mark.parametrize(
+    "input_type, feature_names",
+    [('dataframe', None),
+     ('dataframe', load_boston().feature_names),
+     ('list', load_boston().feature_names),
+     ('array', load_boston().feature_names)]
+)
+def test_plot_partial_dependence_str_features(pyplot, clf_boston, boston,
```

Comment:
Can we do something like to avoid use `load_boston`?

```py
@pytest.mark.parametrize("input_type, use_feature_names", [
	('dataframe', False),
    ('dataframe', True), ...
])
def test_plot_partial_dependence_str_features(..., input_type, use_feature_names):
    feature_names = boston.feature_names if use_feature_names else None
    ...
        
```